### PR TITLE
Make upgraded scripts canonical

### PR DESCRIPTION
**Pull request by joshlory**
This change mirrors https://github.com/code-dot-org/code-dot-org/pull/1295 for the remaining upgraded scripts.

No rush to merge this change.  At minimum, I'll wait to confirm https://github.com/code-dot-org/code-dot-org/pull/1389 looks good in production.

### CODE REVIEW

File: dashboard/test/controllers/scripts_controller_test.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ScriptsControllerTest < ActionController::TestCase
   end
 
   test "should get show of hoc" do
-    get :show, id: Script::HOC_ID
```
Comment (laurelfan) [inline dashboard/test/controllers/scripts_controller_test.rb]: should we also add some tests of the redirects? (test/integrations/redirects_test.rb)

Comment (joshlory) [inline dashboard/test/controllers/scripts_controller_test.rb]: Good call, I'll update.

File: dashboard/app/helpers/levels_helper.rb [hunk 2]
```diff
@@ -4,28 +4,11 @@ module LevelsHelper
 
   def build_script_level_path(script_level)
     if script_level.script.name == 'hourofcode'
```
Comment (joshlory) [inline dashboard/app/helpers/levels_helper.rb]: TODO: Use `Script::HOC_NAME` (same for 'flappy' below).

File: dashboard/test/controllers/scripts_controller_test.rb (hunk 1 above)
Comment (joshlory) [inline dashboard/test/controllers/scripts_controller_test.rb]: Added in ff32db6.

**Pull request merged**